mkosi/config.py

```diff
@@ -2900,17 +2921,18 @@ def root_password_transformer(
             return None
         return (cast(str, rootpw[0]), cast(bool, rootpw[1]))
 
-    def source_target_transformer(
-        trees: list[list[Optional[str]]], fieldtype: type[list[tuple[Path, Path]]]
-    ) -> list[tuple[Path, Path]]:
+    def tree_transformer(trees: list[dict[str, Any]], fieldtype: type[ConfigTree]) -> list[ConfigTree]:
```

Comment:
```suggestion
    def config_tree_transformer(trees: list[dict[str, Any]], fieldtype: type[ConfigTree]) -> list[ConfigTree]:
```
to stay closer to the name